src/test/java/core/basesyntax/repository/StorageImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.repository;
+
+import static core.basesyntax.TestObjects.APPLE;
+import static core.basesyntax.TestObjects.BANANA;
+import static core.basesyntax.TestObjects.ORANGE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageImplTest {
+    private Storage storage;
+
+    @BeforeEach
+    void setUp() {
+        storage = new StorageImpl();
+        storage.getAll().clear();
+    }
+
+    @Test
+    void add_isOk() {
+        storage.add(APPLE, 23);
+        Integer appleQuantity = storage.getQuantity(APPLE);
+        assertEquals(23, appleQuantity);
+    }
+
+    @Test
+    void getQuantity_isOk() {
+        storage.add(BANANA, 11);
+        Integer bananaQuantity = storage.getQuantity(BANANA);
+        assertEquals(11, bananaQuantity);
+    }
+
+    @Test
+    void getQuantity_notOk() {
+        storage.add(APPLE, 23);
+        Integer orange = storage.getQuantity(ORANGE);
+        assertNotEquals(23, orange);
+    }
```

Comment:
This case is rather mistake of usage than development, so you can remove this test

Reply:
Agree, done